### DISCUSSION TITLE
Why was KhronosGroup.VulkanSDK removed from winget?

### DISCUSSION THREAD

**Discussion by Wallby** [microsoft/winget-pkgs#135517] [Q&A] [answered]
I.e. I saw it was missing today and confirmed that it was indeed removed in December..
https://github.com/microsoft/winget-pkgs/commits/master/manifests/k/KhronosGroup
![image](https://github.com/microsoft/winget-pkgs/assets/13614975/b252f024-5c2e-4433-b07a-07d8e73f16a1)

I cannot find any information as to why it was removed.

**Comment by mdanish-kh** [comment, marked as answer]
This is related to:

- https://github.com/microsoft/winget-pkgs/issues/131328
